purview/config.py

```diff
@@ -4,9 +4,8 @@
 
 
 class Config(object):
-    """
-    A Purview Flask configuration
-    """
+    """The base Purview application configuration."""
```

Comment:
Any specific reason to have the quotes and the comment in a single line? If not then maybe we can have a uniform pattern throughout

Reply:
If you have a docstring that is just one line of text, it has to be one line like that based on the PEP guidelines.